### ISSUE TITLE
CTD when entering the Winking Skeever Loft in Solitude

### ISSUE THREAD

**Issue by AELIENUS** [ForgottenGlory/Living-Skyrim-3#612]
**If you are reporting a crash to desktop, please attach your NET Script Framework crash log. This can be found in `[LS Install folder]\mods\Files Generated during playsessions launched with SKSE\NetScriptFramework\Crash`
[Crash_2021_12_11_11-32-35.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7697247/Crash_2021_12_11_11-32-35.txt)

If possible, please also attach a copy of your most recent save before the issue occurred.**

Please check back on your bug request periodically. I will ask for more details sometimes and if they are not provided I will eventually close the bug report.

**LS Version**

Put the version of the list you are on here.
3.6.0.2
**Describe the bug**
CTD when entering the Winking Skeever Loft in Solitude

**To Reproduce**
Enter the Winking Skeever Loft

**Expected behavior**
No CTD

**Screenshots**

If your report pertains to a specific NPC, object, or item, please open the console using the tilde (~) key and then click on that NPC, object or item. Take a screenshot of this and include that in this report. You will know if you have the correct thing selected if it is highlighted white after clicking on it.

**Additional context**

**Comment by AELIENUS** [comment]
Seems to happen only when entering the Winking Skeever Loft during Daytime (ingame).

**Comment by jcalero** [comment]
+1

This happened to me twice in a row today. I had just purchased the loft, went up, spent some time exploring, accepted the quests available inside, left the loft. Everything seemed fine. Went back inside and it crashed to desktop before entering the loft:

Crash 1: [Crash_2022_1_16_14-53-11.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7877222/Crash_2022_1_16_14-53-11.txt)

I reload a save from before purchasing the loft, repeat the process as above but this time I exit through the balcony. As I try to go back in to the loft from the balcony, crash again:

Crash 2: [Crash_2022_1_16_15-7-9.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7877225/Crash_2022_1_16_15-7-9.txt)

---

Edit: The above happened all at night, 3-4am ingame. I now tried a third time and fourth time and I managed to load the loft without crashing. Seems inconsistent, not sure how to reproduce.
